governance-exercises/JanSky Polling Exercise 4 - Deploy Emergency Fix A in imminent executive -  August 9, 2023.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Deploy Emergency Fix A in imminent executive -  August 9, 2023
```

Comment:
Technically, the question is whether governance want Fix A at all (right now, potentially worth re-voting later)

If they do want it, it can  be included in a single executive with Fix B.
Otherwise, single executive with only Fix B.